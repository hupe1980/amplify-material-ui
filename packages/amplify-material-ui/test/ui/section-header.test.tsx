import * as React from 'react';
import { render } from '@testing-library/react';
import { SectionHeader } from '../../src';
import { withTheme } from '../helpers/withTheme';

describe('section-header', () => {
  it('should be rendered correctly', () => {
    const { asFragment } = render(
      withTheme(
        <SectionHeader>
          <div />
        </SectionHeader>,
      )
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
