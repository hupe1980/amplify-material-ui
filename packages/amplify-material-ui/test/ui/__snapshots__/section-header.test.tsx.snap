// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`section-header should be rendered correctly 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-box-1 MuiBox-root css-0"
  >
    <div
      class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault makeStyles-avatar-2 css-2s90m6-MuiAvatar-root"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
        data-testid="LockOutlinedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M18 8h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zM9 6c0-1.66 1.34-3 3-3s3 1.34 3 3v2H9V6zm9 14H6V10h12v10zm-6-3c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"
        />
      </svg>
    </div>
    <h1
      class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
    >
      <div />
    </h1>
  </div>
</DocumentFragment>
`;
